api/types/client.go

```diff
@@ -6,7 +6,9 @@ import (
 	"fmt"
 	"time"
 
+	v12 "k8s.io/api/apps/v1"
```

Comment:
why v12 rather than appsv1?

Reply:
That was my IDE, I have just switched to appsv1 as it really looks way better.